chart/values.yaml

```diff
@@ -77,7 +69,28 @@ prometheus:
     # we provide a config map which will have the
     # timescale-prometheus connector already configured as
     # a remote endpoint for read/write (if enabled)
+    # this overrides the config map the prometheus chart 
+    # creates
     configMapOverrideName: "prometheus-config"
+    # values to be used in the override config map
+    # for a Timescale-Prometheus remote_write and remote_read config
+    timescaleRemote:
```

Comment:
This name threw me off a bit; Given that we're simply trying to forward values to the prometheus chart, could we use their labels and just override those implicitly? I might be missing something:

``
prometheus:
  enabled: true
  alertmanager:
    enabled: false
  pushgateway:
    enabled: false
  server:
     remoteWrite: 
   ### prometheus things?
``

Reply:
It's not impossible, just that we'd have some more templating logic.
1. Current implementation for `server.remoteWrite` (if we keep timescaleRemote separate)
  - server.remoteWrite | toYaml | indent
2. If we were to make the url templated
  - writeConfig := range server.remoteWrite
     - key, value := range  $writeConfig
     - if $key == url then $key: {{ tpl $value }}
     - else $key: ($value | toYaml)
and I don't know if I can manage to align the properties for $key: ($value | toYaml)
because they can sometimes be simple values, but also maps

and then repeat it for remote_read